website/src/components/Sponsor/index.tsx

```diff
@@ -23,19 +23,23 @@ const Sponsor: React.FC = () => {
           <Grid container>
             <Grid item xs={12} md={6} sm={6}>
               <Typography className={classes.paragraph}>{t("sponsors.mayadata")}</Typography>
-              <img
-                src="../images/logos/mayadata_logo.svg"
-                alt={t("generic.mayadata")}
-                className={classes.sponsorCompany}
-              />
+              <a href="https://mayadata.io/" rel="noreferrer" target="_blank">
```

Comment:
Please take `https://mayadata.io/` and `https://www.cncf.io/` from constants file

Reply:
<Link /> from `material-ui` can be used for external links also

Reply:
Let's use the Link element

Reply:
Done